CV-HTML-CSS/style.css

```diff
@@ -0,0 +1,149 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  height: inherit;
+  font-family: "Lato", Arial, serif !important;
+}
+
+h2 {
+  color: rgb(56, 56, 56);
```

Comment:
in other places u use HEX notation, so use it here too, because your document have to look like the same in
different parts